assets/app/views/events.html

```diff
@@ -39,9 +39,13 @@
                     <td class="hidden-sm hidden-md" data-title="Kind">
                       {{event.involvedObject.kind}}</td>
                     <td class="hidden-sm hidden-md" data-title="Reason">
+                      <event-icon type="{{event.type}}"></event-icon>
```

Comment:
Can the browse -> events page reuse the directive? If kind and name aren't set in the directive, show all events?